pool/src/pool/actions.rs

```diff
@@ -133,6 +134,7 @@ pub fn build_actions_from_request(
             }
             RequestType::Withdraw => {
                 let mut reserve = pool.load_reserve(e, &request.address, true);
+                reserve.require_action_allowed(e, request.request_type);
```

Comment:
withdrawing assets is always allowed, regardless of any combination of pool or reserve status, thus this call is not needed